commit 047f18d2a9f6c8a61f164db043bc65591e5aaa4e (refs/changes/22/522/3)
Author: Trent Mick <trentm@gmail.com>
Date:   2016-09-22T15:42:19-07:00 (3 years, 1 month ago)
    
    CAPI-523 add contribution blurb to node-ufds' README
    Reviewed by: Patrick Mooney <patrick.mooney@joyent.com>
    Approved by: Patrick Mooney <patrick.mooney@joyent.com>

diff --git a/CHANGES.md b/CHANGES.md
new file mode 100644
index 0000000..901da0e
--- /dev/null
+++ b/CHANGES.md
@@ -0,0 +1,9 @@
+# node-ufds changelog
+
+## 1.2.0
+
+- Add support for a clean install with node v4 (CAPI-522).
+
+## earlier version
+
+Use the commits, Luke.
diff --git a/README.md b/README.md
index 6255d06..fda4eeb 100644
--- a/README.md
+++ b/README.md
@@ -5,19 +5,18 @@
 -->
 
 <!--
-    Copyright (c) 2014, Joyent, Inc.
+    Copyright 2016 Joyent, Inc.
 -->
 
-# Node.js Clients for SmartDataCenter UFDS
+# node-ufds
 
-This repository is part of the Joyent SmartDataCenter project (SDC).  For
-contribution guidelines, issues, and general documentation, visit the main
-[SDC](http://github.com/joyent/sdc) project page.
+This repository is part of the Joyent Triton project. See the [contribution
+guidelines](https://github.com/joyent/triton/blob/master/CONTRIBUTING.md) --
+*Triton does not use GitHub PRs* -- and general documentation at the main
+[Triton project](https://github.com/joyent/triton) page.
 
-
-# Overview
-
-Node.js client library for SDC UFDS service.
+This is a Node.js client library for Triton's
+[UFDS](https://github.com/joyent/sdc-ufds) service.
 
 
 # Development
@@ -28,7 +27,4 @@ the test suites.
 
 # Testing
 
-Short version:
-
     make test
-
diff --git a/package.json b/package.json
index 9fa1a88..75f5cdb 100644
--- a/package.json
+++ b/package.json
@@ -2,7 +2,7 @@
   "name": "ufds",
   "author": "Joyent (joyent.com)",
   "description": "SmartDataCenter UFDS Client API",
-  "version": "1.1.3",
+  "version": "1.2.0",
   "homepage": "https://github.com/joyent/sdc",
   "repository": {
     "type": "git",
